packages/block-library/src/list-item/hooks/use-enter.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createBlock,
+	getDefaultBlockName,
+	cloneBlock,
+} from '@wordpress/blocks';
+import { useRef } from '@wordpress/element';
+import { useRefEffect } from '@wordpress/compose';
+import { ENTER } from '@wordpress/keycodes';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import useOutdentListItem from './use-indent-list-item';
+
+export default function useEnter( props ) {
+	const { replaceBlocks } = useDispatch( blockEditorStore );
+	const {
+		getBlock,
+		getBlockRootClientId,
+		getBlockParents,
+		getBlockIndex,
+	} = useSelect( blockEditorStore );
+	const propsRef = useRef( props );
+	propsRef.current = props;
+	const [ canOutdent, outdentListItem ] = useOutdentListItem(
+		propsRef.current.clientId
+	);
+	return useRefEffect(
+		( element ) => {
+			function onKeyDown( event ) {
+				if ( event.defaultPrevented || event.keyCode !== ENTER ) {
+					return;
+				}
+				const { content, clientId } = propsRef.current;
+				if ( content.length ) {
+					return;
+				}
+				event.preventDefault();
+				if ( canOutdent ) {
+					outdentListItem();
```

Comment:
Shouldn't this be indent?

Reply:
Why? It's working as expected in the video, no?

Reply:
For me it was indenting for some reason instead of outdenting.